esmvaltool/diag_scripts/ocean/diagnostic_maps.py

```diff
@@ -39,6 +47,37 @@
 logging.getLogger().addHandler(logging.StreamHandler(sys.stdout))
 
 
+def regrid_irregulars(cube, scheme='nearest'):
```

Comment:
This function looks pretty similar to `esmvaltool.preprocessor.regrid`, wouldn't it be possible to use that?

Reply:
It's still here, did you commit and push your changes?

Reply:
Yes, it is gone: https://github.com/ESMValGroup/ESMValTool/blob/version2_development_ocean_documentation/esmvaltool/diag_scripts/ocean/diagnostic_maps.py